spec/timezone.html

```diff
@@ -362,12 +401,83 @@ <h1>CreateTemporalTimeZone ( _identifier_ [ , _newTarget_ ] )</h1>
       <emu-alg>
         1. Assert: ! CanonicalizeTimeZoneName(_identifier_) is _identifier_.
         1. If _newTarget_ is not given, set it to %Temporal.TimeZone%.
-        1. Let _object_ be ? OrdinaryCreateFromConstructor(_newTarget_, *"%Temporal.TimeZone.prototype%"*, « [[InitializedTemporalTimeZone]], [[Identifier]] »).
+        1. Let _object_ be ? OrdinaryCreateFromConstructor(_newTarget_, *"%Temporal.TimeZone.prototype%"*, « [[InitializedTemporalTimeZone]], [[Identifier]], [[IsOffset]], [[OffsetNanoseconds]] »).
         1. Set _object_.[[Identifier]] to _identifier_.
+        1. If _identifier_ satisfies the syntax of a |TimeZoneNumericUTCOffset| (see <emu-xref href="#sec-temporal-iso8601grammar"></emu-xref>), then
+          1. Set _object_.[[IsOffset]] to *true*.
+          1. Set _object_.[[OffsetNanoseconds]] to ! ParseTimeZoneOffsetString(_identifier_).
+        1. Else,
+          1. Set _object_.[[IsOffset]] to *false*.
+          1. Set _object_.[[OffsetNanoseconds]] to *undefined*.
```

Comment:
Do you need both internal slots, or does "[[OffsetNanoseconds]] is not **undefined**" work?

Reply:
That works just as well; I was thinking that this might be more understandable, but I've changed it.